Mta 4707 (#3300)
scaleway/docs-content: tutorials/easydeploy-grafana/index.mdx

---
meta:

>   title: Deploy ArgoCD on Scaleway Kubernetes clusters using Easy Deploy

Reviewer: Metadata are wrong, this page is about Grafana

Author: ```suggestion
  title: Deploy Grafana on Scaleway Kubernetes clusters using Easy Deploy
```

>   description: Discover how to use Grafana on Scaleway Kubernetes clusters with Easy Deploy. Unlock the power of data visualization with Grafana, and follow our step-by-step instructions for deployment and management of your application.

Author: ```suggestion
  description: Use Grafana on Scaleway Kubernetes clusters with Easy Deploy. Unlock data visualization power and follow our step-by-step instructions for deployment and management.
```

content:

>   h1: Deploy ArgoCD on Scaleway Kubernetes clusters using Easy Deploy

Reviewer: H1 title is wrong, this tutorial is about Grafana

Author: ```suggestion
  h1: Deploy Grafana on Scaleway Kubernetes clusters using Easy Deploy
```

>   paragraph: Discover how to use Grafana on Scaleway Kubernetes clusters with Easy Deploy. Unlock the power of data visualization with Grafana, and follow our step-by-step instructions for deployment and management of your application.

Author: ```suggestion
  paragraph: Use Grafana on Scaleway Kubernetes clusters with Easy Deploy. Unlock data visualization power and follow our step-by-step instructions for deployment and management.
```

- Created a Scaleway Kubernetes [Kapsule](/containers/kubernetes/how-to/create-cluster/) or [Kosmos](/containers/kubernetes/how-to/create-kosmos-cluster/) cluster

## Deploying the Grafana application using Easy Deploy

<Message type="important">
When configuring your Kubernetes cluster, note that certain applications and services, such as Grafana, may result in additional resources being provisioned.
For example, your YAML configuration might create Block Storage volumes or Load Balancers, which could incur extra costs.
</Message>

1. In the [Scaleway console](https://console.scaleway.com/), navigate to the **Kubernetes** section under **Containers**.
2. Click the name of the cluster where you wish to deploy Grafana. The **Cluster Information** tab will display.
3. Click the **Easy Deploy** tab. The application dashboard displays.
4. Click **Deploy Application**. The application deployment wizard displays.
5. Choose **Application Library** to see the list of available applications.
6. Select the **Grafana** application.
    <Message type="tip">
        If you cannot find Grafana on the first page, use the search bar or navigate through the library.
    </Message>

> 7. Optionally, customize the default configuration for Grafana using [Helm Charts](/tutorials/kubernetes-package-management-helm/). If you do not require any customized configuration you can skip this step.

Reviewer 2: ```suggestion
7. Optionally, customize the default configuration for Grafana using [Helm Charts](/tutorials/kubernetes-package-management-helm/). If you do not need any customized configuration you can skip this step.
```

8. Enter a name and a Kubernetes namespace for your application. If no name is entered, Grafana will be installed in the default namespace of the cluster.
9. Click **Deploy Application** to deploy Grafana on your Kubernetes cluster.

## Retrieving the Grafana admin password

1. Access the [Kubernetes Dashboard](/containers/kubernetes/how-to/access-kubernetes-dashboard/) of your cluster and navigate to **Secrets** under the **Config and Storage** section.
2. Select the name of your Grafana application to display its data.

> 3. Click the "eye" icon next to **admin-password** to reveal the Grafana admin password. Copy the password.

Reviewer 3: ```suggestion
3. Click the eye icon next to **admin-password** to reveal the Grafana admin password. Copy the password.
```

## Connecting to Grafana

> Grafana uses a LoadBalancer to manage traffic. To check the deployment status, run:

Reviewer 2: ```suggestion
Grafana uses a Load Balancer to manage traffic. To check the deployment status, run:
```

```
$ kubectl get svc
NAME                               TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)        AGE
application-silly-sammet-grafana   LoadBalancer   10.34.223.177   51.159.xxx.yy   80:32483/TCP   35s
kubernetes                         ClusterIP      10.32.0.1       <none>          443/TCP        4h58m
```

Once the external IP status changes from `<PENDING>` to the assigned external IP, your Grafana application is up. Copy the IP address and paste it into a web browser. The Grafana login screen will appear.

## Logging into Grafana

1. Paste the external IP address of the LoadBalancer service into your web browser to open the Grafana login screen.

> 2. Enter **admin** as the username and the password retrieved in the previous step as the password to log in.

Reviewer 2: ```suggestion
2. Enter **admin** as the username and the password retrieved in the previous step to log in.
```
